src/react-apps/applications/shared/src/version-control/versionControlHeader.tsx

```diff
@@ -161,7 +161,7 @@ class VersionControlHeader extends React.Component<IVersionControlHeaderProps, I
 
     } catch (err) {
       if (axios.isCancel(err)) {
-        // console.error('Component did unmount. Get canceled.');
+        // console.info('Component did unmount. Get canceled.');
```

Comment:
It's still commented out. Maybe change out the if for `!axios.isCancel`? So we log when it fails? And remove `else`.

Reply:
Commented the handy debug line.